cookbooks/travis_internal_base/spec/ssh_spec.rb

```diff
@@ -24,35 +23,42 @@
   gssapistrictacceptorcheck yes
   hostbasedauthentication no
   hostbasedusesnamefrompacketonly no
-  hostkey /etc/ssh/ssh_host_ed25519_key
   hostkey /etc/ssh/ssh_host_rsa_key
+  hostkey /etc/ssh/ssh_host_dsa_key
+  hostkey /etc/ssh/ssh_host_ecdsa_key
+  hostkey /etc/ssh/ssh_host_ed25519_key
+  acceptenv LANG
+  acceptenv LC_*
+  subsystem sftp /usr/lib/openssh/sftp-server
```

Comment:
IIRC I intentionally left this list incomplete to protect the spec from failing on distro differences that I considered low-risk, given that the assertion is that the output is a superset of the values here.